README.md

```diff
@@ -220,6 +220,13 @@ WebChat currently defaults to *not* sending 'typing' activities to the bot when
 
 You can supply WebChat with the id (and, optionally, a friendly name) of the current user by passing `user: { id: user_id, name: user_name }` to `App`/`Chat`. This object is passed with every activity sent from WebChat to the bot, which means it is not available to the bot *before* any activities are sent. See [The Backchannel](#the-backchannel) to find out how your web page can programmatically send non-message activities to the bot.
 
+### Provide Chat History
+
+You can initialize the WebChat conversation by passing an array of activity objects in `pastHistory: [ activity_object1, activity_object2, ... ]` to `App`/`Chat`. Each activity object should have a unique `id` value, and the `from` property should match either a user identity (above) or the bot's identity. An example activity message might look like:
+```
```

Comment:
I'm worried about identity. You say the user and bot identities should match up, but you don't check for it. Also, this history might include identities of other folk in a conversation. Here's one idea: what if in this list of activities you have special tokens for identities, e.g. from: { id: "$BOT$" } and from : {ID: "$USER$"} which we would automatically replace with the current bot and user identities as supplied (including the name)? Others would be left alone.

If you don't do this, I think you should supply a little more guidance on how they should go about supplying a well-formed history, which should probably go into how to store the history. It is at the point the history is originally created that they know who is who.